packages/pwa-kit-dev/src/ssr/server/loading-screen/index.html

```diff
@@ -87,7 +88,7 @@ <h1 class="loading-text">Building your app</h1>
 
                 waitForReady().then(() => {
                     // Redirect to homepage
-                    window.location = url.origin
+                    window.location = `${url.origin}?loaded=${projectName}`
```

Comment:
FYI I decided to replace one of my previous solutions, the `Clear-Site-Data` header, with this cache breaker method instead. If our criteria are these, then I feel that the cache breaker is better.
- minimal, non-intrusive: developers still have control
- visible: developers notice it and know what’s causing it

Using the `Clear-Site-Data` header would clear the cache for all pages under the origin. But now I feel that it's intrusive. And it may not be clear to other developers that this is happening. Furthermore, there is no way for them to turn it off, as the logic lies in the dev package, and clearing the cache also adds a slight delay to the rendering of the loading page.

On the other hand, the cache breaker method also have some cons but they're not that bad IMO. First of all, the cache breaker is applied only for the homepage url. But for example, `?loaded=retail-react-app`, it's going to be clearer for the developers that the loading page is the one doing that. And again, there's no way to turn it off, because the logic lies in the dev package. But the developers can work around that by manually navigating to the homepage.

Reply:
Why was the cache breaker has the name of the project (and not some timestamp or random strings)?
- Because it's meant to avoid stale cache when navigating between projects (and not within an individual project).
- If  developers want to test their cache header within a project, they can still do that.

Reply:
As per John's feedback, I've added timestamp as suffix for our cache breaker. So that it'll break cache for when you're in the same project but have switched branches.